src/hiveboard_bridge/src/main.cpp

```diff
@@ -47,7 +44,12 @@ int main(int argc, char** argv) {
         // Publish on moveby
         moveByPublisher.publish(moveByMessage);
 
-        // TODO Send ack/response over TCP
+        // Send ack/response
+        FunctionCallResponseDTO functionCallResponse(GenericResponseStatusDTO::Ok, "moveBy OK");
+        UserCallResponseDTO userCallResponse(UserCallDestinationDTO::BUZZ, functionCallResponse);
+        ResponseDTO response(1, userCallResponse);
```

Comment:
The ID of the response should be the same of the request, so the caller can match them